python/cudf/requirements/cuda-10.1/requirements-run.txt

```diff
@@ -0,0 +1,10 @@
+cupy-cuda101
+fastavro>=0.22.9
+notebook>=0.5.0
+cython>=0.29,<0.30
+fsspec>=0.6.0
+ipython
+partd
+dask>=2.22.0
+distributed>=2.22.0
+streamz
```

Comment:
Is this supposed to be on top of the build requirements? I think we can safely remove `notebook`, `ipython`, `partd`, `dask`, `distributed`, and `streamz` from here as those are for `dask-cudf` / `custreamz` which should land in a separate place.